gridstatus/isone_api/isone_api.py

```diff
@@ -231,9 +232,6 @@ def get_realtime_hourly_demand(
                         )
                     url = f"{BASE_URL}/realtimehourlydemand/current/location/{location_id}"
                     response = self.make_api_call(url)
-                    from pprint import pprint
-
-                    pprint(response)
```

Comment:
accidently left this in last time. I have an idea of a better way to create fixtures in-situ now as well.